### PR TITLE
fix: Fix FeedView template not loading on windows

### PR DESCRIPTION
**Pull request by dr1rrb**
## Bugfix
Fix FeedView template not loading on windows

fixes regression intoduced by https://github.com/unoplatform/uno.extensions/pull/1239

## What is the current behavior?
App crash at startup on windows platform

## What is the new behavior?
🙃

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

### CODE REVIEW

**Review by agneszitte-nventive — APPROVED** [review]

File: src/Uno.Extensions.Reactive.UI/Uno.Extensions.Reactive.WinUI.csproj [hunk 1]
```diff
@@ -16,6 +16,19 @@
...
 	<Import Project="common.props" />
 
+	<ItemGroup>
+		<Page Update="Themes\Generic.WinUI.xaml" Link="Themes\Generic.xaml" />
+		<Page Remove="Themes\Generic.UI.xaml" />
+		<None Include="Themes\Generic.UI.xaml" />
+	</ItemGroup>
+
+	<Target Name="_RenameGenericXamlFile" BeforeTargets="_CopyOutOfDateSourceItemsToOutputDirectory">
+		<ItemGroup>
```
Comment (dr1rrb) [inline src/Uno.Extensions.Reactive.UI/Uno.Extensions.Reactive.WinUI.csproj:26]:
```suggestion
		<!-- The generic.xaml file is referenced using a Link, but XAML and XBF files generated by the MarkupCompilePass1 are still named generic.WinUI.<xaml|xbf>, which causes the _CopyOutOfDateSourceItemsToOutputDirectory to fail. -->
		<ItemGroup>
```

File: src/Uno.Extensions.Reactive.UI/Uno.Extensions.Reactive.UI.csproj [hunk 2]
```diff
@@ -13,6 +13,12 @@
 	</PropertyGroup>
 	
 	<Import Project="common.props" />
+	
+	<ItemGroup>
+		<Page Update="Themes\Generic.UI.xaml" Link="Themes\Generic.xaml" />
+		<Page Remove="Themes\Generic.WinUI.xaml" />
+	  <None Include="Themes\Generic.WinUI.xaml" />
+	</ItemGroup>
```
Comment (dr1rrb) [inline src/Uno.Extensions.Reactive.UI/Uno.Extensions.Reactive.UI.csproj:22]:
```suggestion

	<Target Name="_RenameGenericXamlFile" BeforeTargets="_CopyOutOfDateSourceItemsToOutputDirectory">
		<!-- The generic.xaml file is referenced using a Link, but XAML and XBF files generated by the MarkupCompilePass1 are still named generic.UI.<xaml|xbf>, which causes the _CopyOutOfDateSourceItemsToOutputDirectory to fail. -->
		<ItemGroup>
dr1rrb marked this conversation as resolved.
			<GenericXamlOutput Include="$(IntermediateOutputPath)\Themes\Generic.UI.*" />
		</ItemGroup>
		<Copy SourceFiles="@(GenericXamlOutput)" DestinationFiles="@(GenericXamlOutput->Replace('Generic.UI', 'Generic'))" />
	</Target>
```

**Comment by nickrandolph** [comment]
@dr1rrb I think this actually highlights a bit of an issue with some of the extensions libraries. Currently there's a mix of some libraries where the WinUI version has the same assembly name as the UI version (eg Uno.Extensions.Navigation.UI is the assembly for both WinUi and UI versions) and some libraries where the assembly ends in WinUI.
My suggestion would be for us to follow what Uno.UI/Uno.WinUI does which is that the package **only** is differentiated by the WinUI extensions and that the assemblies all use .UI

**Comment by dr1rrb** [comment]
> @dr1rrb I think this actually highlights a bit of an issue with some of the extensions libraries. Currently there's a mix of some libraries where the WinUI version has the same assembly name as the UI version (eg Uno.Extensions.Navigation.UI is the assembly for both WinUi and UI versions) and some libraries where the assembly ends in WinUI. My suggestion would be for us to follow what Uno.UI/Uno.WinUI does which is that the package **only** is differentiated by the WinUI extensions and that the assemblies all use .UI

@nickrandolph well actually it was a mistake I made ... the assembly name is the same, so I do have to remove the magic rename I did yesterday

**Pull request merged**